### ISSUE TITLE
training loss always `nan`

### ISSUE THREAD

**Issue by fhrzn** [seongminp/transformers-into-vaes#4]
Hello, first of all thanks for your amazing work!
I am trying to barely run your code in Google Colab but got the training loss always nan when starting pretraining encoder. Among 15 epochs, it just stop after 1st epoch as the EarlyStopCallback noticed that the loss keep infinite since the beginning. If it possible, could you please check to reproduce my errors?

My environment:
- Transformers  ==  4.23.1
- Pytorch Lightning  ==  1.7.7

Looking forward for your response, thank you)

**Comment by fhrzn** [comment]
Found the issue, there were a mismatch on assigning variables at denoising input function.

Need to modify `run_batch()` in the file `model_t5.py`. Go to line 100 and 101, then change these lines
```python3
encoder_inputs[i] = msk
encoder_masks[i] = inp
```

into

```python3
encoder_inputs[i] = inp
encoder_masks[i] = msk
```

**Comment by seongminp** [comment]
Oh my.. thank you for finding this bug. Really appreciate it!
